rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -6551,6 +6552,312 @@ private void assertRefundInProcessPegInVersionLegacy(
         Assert.assertFalse(provider.getHeightIfBtcTxhashIsAlreadyProcessed(btcTx.getHash()).isPresent());
     }
 
+    private Address getFastBridgeAddressFromRedeemScript(Script redeemScript, Sha256Hash derivationArgumentHash) {
+        Script fastBridgeRedeemScript = FastBridgeRedeemScriptParser.createMultiSigFastBridgeRedeemScript(
+            redeemScript,
+            derivationArgumentHash
+        );
+
+        Script fastBridgeP2SH = ScriptBuilder.createP2SHOutputScript(fastBridgeRedeemScript);
+        return Address.fromP2SHScript(bridgeConstants.getBtcParams(), fastBridgeP2SH);
+    }
+
+    private BigInteger registerFastBridgeBtcTransaction_RSKIP293(
+        ActivationConfig.ForBlock activations,
```

Comment:
```suggestion
        boolean isRskip293Active,
```
This could simplify a bit. Then inside the method 
```
ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
when(activations.isActive(ConsensusRule.RSKIP176)).thenReturn(true);
when(activations.isActive(ConsensusRule.RSKIP219)).thenReturn(true);
when(activations.isActive(ConsensusRule.RSKIP293)).thenReturn(isRskip293Active);
```